### DISCUSSION TITLE
Blueprint UI Design Tool Resources

### DISCUSSION THREAD

**Discussion by alexofthewu** [palantir/blueprint#5743] [Ideas]
#### Environment

- __Package version(s)__: 
3.0
- __Browser and OS versions__: 
Chrome, macOS 10
#### Question
Hey Blueprint team,

A project I'm working on is looking to use Blueprint UI for the design system. 

Will there be a Sketch or Figma update for the resources for 3.0? I'm happy to help, wondering what resource roadmaps you have or plans to improve the Sketch file. We use Figma, so I'm happy to help in that case.

Looks like the last update was August 2018.

**Comment by adidahiya** [comment]
The sketch file is up to date with Blueprint 3.0.

The latest discussion and work around the design resources can be found here and in the linked PRs issues on this thread: https://github.com/palantir/blueprint/issues/2817

**Comment by kudigaracoder** [comment]
I'd be interested in contributing to the Figma Design resource if there's ever a plan to build one.

**Comment by kudigaracoder** [comment]
@alexofthewu I've started building out a Figma version of Blueprint. Let me know if you would like to help/contribute?

**Comment by dmackerman** [comment]
@kudigaracoder Interested. How would we collaborate on this Figma file?

**Comment by kudigaracoder** [comment]
@dmackerman ~We would have to set up guidelines before as everyone getting on the same file could get confusing.~ I can send out an invite to the file & you can start right away. It'd be good to discuss the component architecture before further building them out?

**Comment by terpimost** [comment]
@kudigaracoder @dmackerman I just finished Sketch version improvement and would like to observe progress on Figma. Please share with me too terpimost@gmail.com

**Comment by dmackerman** [comment]
@kudigaracoder That sounds good. I don't have much time to contribute, but I'd like to take a look at where you landed.

**Comment by jaxgeller** [comment]
@kudigaracoder did this get anywhere? Would love to see this as a community Figma file!

**Comment by atnickallen** [comment]
@kudigaracoder Also am looking for this and would be happy to contribute

**Comment by kudigaracoder** [comment]
Sorry I'm late in responding.. been busy with work. I was in the process of building the Figma library(Colors/Typography/Button/Input) when I got busy with work. After they announced Variants, waited for it to be released to incorporate the feature before proceeding with the other components. I've been working on incorporating variants. I'll try and push to release this ASAP.

**Comment by terpimost** [comment]
Hey @kudigaracoder  Is there any progress on Figma? If somebody share their figma file we could collaborate and do that faster. Anyway I think we could open a new issue about it

**Comment by zachkirsch** [comment]
@adidahiya does Palantir have a Figma library for blueprint that can be shared?

**Comment by mattobee** [comment]
Anyone aware of any progress with this? I'm about to port a Blueprint project from Sketch to Figma. Checking I'm not missing an existing resource before I start from scratch.

**Comment by adidahiya** [comment]
We have a Figma library with most of the components ported over from Sketch. We're hoping to open source that in the next few months!

**Reply by jbsouvestre** [reply]
Hi @adidahiya, is there an update on this by any chance ? Many thanks !

**Comment by ACallander** [comment]
Is any update on this Figma file or on the community Figma being posted soon?

**Comment by zachkirsch** [comment]
+1 would love a Figma file 🙏
